physionet-django/project/models.py

```diff
@@ -1,8 +1,10 @@
+from datetime import datetime
 import hashlib
 import os
 import shutil
 import uuid
 import pdb
+import pytz
```

Comment:
Is `pytz.timezone(timezone.get_default_timezone_name()).localize(dt)` setting the time to the server timezone? Is there a reason why we can't just specify the timezone in `settings.py`?

Reply:
`timezone.get_default_timezone_name()` is reading the timezone from the settings file. pytz.timezone() is then using that to add a time zone to the dt object.